WebDriverAgentTests/IntegrationTests/FBPasteboardTests.m

```diff
@@ -61,15 +61,21 @@ - (void)testGetPasteboard
                                 matchingIdentifier:@"Select All"].firstMatch;
   XCTAssertTrue([selectAllItem waitForExistenceWithTimeout:5]);
   [selectAllItem tap];
-  [textField pressForDuration:2.0];
+  if (SYSTEM_VERSION_LESS_THAN(@"16.0")) {
+    [textField pressForDuration:2.0];
+  }
   XCUIElement *copyItem = [[self.testedApplication descendantsMatchingType:XCUIElementTypeAny]
                            matchingIdentifier:@"Copy"].firstMatch;
   XCTAssertTrue([copyItem waitForExistenceWithTimeout:5]);
   [copyItem tap];
   FBWaitExact(1.0);
   NSData *result = [FBPasteboard dataForType:@"plaintext" error:&error];
   XCTAssertNil(error);
-  XCTAssertEqualObjects(textField.value, [[NSString alloc] initWithData:result encoding:NSUTF8StringEncoding]);
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"16.0")) {
+    // Pasteboard permission appears in a simulator. Not in a real device.
```

Comment:
cannot we just accept the permission then?

Reply:
Maybe we need to add allowing permission popup in the WDA code side. I'll take a look later (but not this PR at this time). This was only for simulator behavior.